### PR TITLE
tools: Add Vulkan SC variant of vulkaninfo

### PR DESCRIPTION
**Pull request by aqnuep**

### CODE REVIEW

File: vulkaninfo/vulkaninfo.h [hunk 1]
```diff
@@ -666,23 +729,27 @@ struct AppInstance {
...
+            std::cerr << "This problem is often caused by a faulty installation of the " API_NAME " driver or attempting to use a GPU "
+                         "that does not support " API_NAME ".\n";
             THROW_VK_ERR("vkCreateInstance", err);
         } else if (err) {
             THROW_VK_ERR("vkCreateInstance", err);
         }
         ext_funcs.LoadInstanceExtensionDispatchPointers(dll.fp_vkGetInstanceProcAddr, instance);
 
+#ifndef VULKANSC
         err = ext_funcs.vkCreateDebugReportCallbackEXT(instance, &dbg_info, nullptr, &debug_callback);
```
Comment (dgkoch) [inline vulkaninfo/vulkaninfo.h:742]: dbg_info is still unconditionally chained into VkInstanceCreateInfo above here - this is an unknown structure in VKSC.

File: output.txt [hunk 2]
```diff
@@ -0,0 +1,1120 @@
+============
+VULKANSCINFO
+============
+
+Vulkan SC Instance Version: 1.0.13
```
Comment (dgkoch) [inline output.txt:5]: Good to see the sample output, but is this file supposed to be checked in?

**Review by dgkoch — CHANGES_REQUESTED** [review]

**Comment by sricker-nvidia** [comment]
I tested on linux after fixing the vkCreateDebugReportCallbackEXT issue mentioned by Daniel, but hit another issue where the VkDeviceCreateInfo structure is missing mandatory structures like VkDeviceObjectReservationCreateInfo in its pNext chain. Haven't investigate further past that issue.

**Comment by dgkoch** [comment]
@aqnuep seems like you need to run this with VVL :-)

**Comment by aqnuep** [comment]
> @aqnuep seems like you need to run this with VVL :-)

Indeed. I guess my validation setup when running it was incorrect. I'll re-check everything.

**Comment by aqnuep** [comment]
@sricker-nvidia, could you give it a go with the updates? It is now certainly validation-clear.

**Review by dgkoch — APPROVED** [review]

**Comment by dgkoch** [comment]
LGTM pending Sam's testing results.

**Comment by sricker-nvidia** [comment]
Just finished testing the latest version of the patch on L4T / embedded linux and QNX. Working as expected on both.

LGTM

**Comment by aqnuep** [comment]
Thanks, @sricker-nvidia!

**Pull request merged**